### ISSUE TITLE
Use stagging letsencrypt endpoint 

### ISSUE THREAD

**Issue by doanguyen** [simple-login/simplelogin-postfix-docker#2]
Instead of production endpoint, which will be throttled

**Comment by nguyenkims** [comment]
@doanguyen yeah using the staging endpoint during the development is a good idea 👌! Maybe this can be set via a environment variable like `TESTING`?

**Comment by doanguyen** [comment]
sure, at the moment, the acme-challenge is done during the build phase in the `Dockerfile`, we will need to move it to the entrypoint instead.

Ref: 
0. acme stagging server: https://certbot.eff.org/docs/using.html#changing-the-acme-server

**Comment by rehanone** [comment]
Closing this issue as per the discussion on #4, it is not required/planned.
